tests/unit/schema/helpers.js

```diff
@@ -8,3 +10,134 @@ export const validManifest = {
     },
   },
 };
+
+/*
+ background: {
+        anyOf: [
+          {
+            type: "object",
+            max_manifest_version: 2,
+            properties: {
+              page: {
+                "$ref": "manifest#/types/ExtensionURL"
+              } 
+            },
+            required: ["page"],
+          },
+          {
+            type: "object",
+            properties: {
+              service_worker: {
+                allOf: [
+                  { "$ref": "manifest#/types/ExtensionURL" },
+                  { min_manifest_version: 3 },
+                ]
+              },
+            },
+            required: ["service_worker"],
+          },
+        ],
+      },
+      */
```

Comment:
If this stays in the file, we should reformat it a bit and maybe add a comment above it.

Reply:
ehm... that was actually a fragment of schema data that I kept around temporarily as a reference while I was working on some of those tests that use custom schema data, I should have removed it but I forgot and got into one of the patches. I'll remove it in the next update.